### ISSUE TITLE
Shippo does not work with newly created products

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2958]
### Expected behavior
Should receive shipping quotes

### Actual Behavior
Receive no shipping quotes

### Steps to Reproduce the Behavior
1. Login as administrator
1. Enable Shippo and enable carriers
1. Create a product (don't use the built-in product)
1. Add **only** the newly created product to cart
1. Attempt to checkout
1. Observe that "No shipping methods are configured. Configure now." is displayed

*Note* this happens on both `master` and `marketplace`. I believe this is because the shippo method is expecting a `parcel` object on the product but it's stored on the variant. This works with the "Basic Reaction Product" because it's coded into the JSON.

### Versions
```
Node: 7.10.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.12.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.06.2-ce
```

**Comment by zenweasel** [comment]
@spencern / @aaronjudd  Not sure if this is something we need to triage first, but it seems like `marketplace` blocker?

**Comment by spencern** [comment]
Does seem like this could be a blocker. Even if it's not a regression, it should probably be resolved asap.

**Comment by zenweasel** [comment]
So I think the solution is simple but twofold.

1. Restore/re-enable setting dimensions/weight at the product level on the PDP (you can currently only set it on the top-level variant)
1. Have shippo use either the variant or the product dimensions/weight when getting quotes for Shippo

**Comment by zenweasel** [comment]
So it actually looks like you can't even set parcel at the variant level. When you set them you get `height`, `weight`, etc. on the top level variant. Not sure where `parcel` is coming from.

**Comment by spencern** [comment]
This is something that @rymorgan and I have discussed in https://github.com/reactioncommerce/reaction/pull/2893#issuecomment-332406765 a little bit.

**Comment by rymorgan** [comment]
@spencern I'm not sure I'm following exactly what's going on here. I definitely see that I'm getting the no shipping methods configured when I follow the steps outlined by @zenweasel -- if we need to allow/force height weight etc at option level to get this to work, then we can just add those fields. We can figure out a better UX down the line if that's the case.

**Comment by spencern** [comment]
Resolved by #2992
